.gitlab-ci.yml

```diff
@@ -83,6 +83,33 @@ build-spack-nightlies:
         - spack install --test=root --reuse
 
 
+### Test lcg release in spack  on the key4hep infrastructure
+build-lcg-release:
+    stage: compilation
+    needs:
+        - "setup_spack_push"
+    only:
+      refs:
+          - schedules # Only execute this on scheduled "nightly" pipelines
+    script:
+        - source spack/share/spack/setup-env.sh
+        - git clone https://gitlab.cern.ch/sft/sft-spack-repo.git
+        - cd sft-spack-repo
+        - source environments/lcg-release/setup_clingo_centos7.sh
```

Comment:
```suggestion
```
The build machine is already setup with clingo. Not putting this here helps with using this CI job on other platforms

Reply:
Yes, please add the `k4-build-spack-release` tag and make the job conditional on `$K4_JOBTYPE == "LCG_Release"` for example